scripts/helper.js

```diff
@@ -229,6 +230,7 @@ const accountsMetadata = (network) => {
       pool: contract('pool.seeds', 'pool'),
       dao: contract('dao.seeds', 'dao'),
       startoken: contract('star.seeds', 'startoken'),
+      tokensmaster: contract('modls.seeds', 'tokensmaster'),
```

Comment:
What's modls stand for? Haha. If this is a generic contract, we may just use "tokenmasterx" or whatever. 

I mean if we expect people other than Seeds using it. 

tmastr.seeds ?  

tmgr.seeds ?

Reply:
I made it `tmastr.seeds`